.github/workflows/update-openebs.yaml

```diff
@@ -18,10 +18,50 @@ jobs:
         go-version: 1.19
 
     - name: Create OpenEBS 2.x Update
+      id: update
       working-directory: ./addons/openebs/template
       run: ./generate.sh --version=^2.0.0
 
+    - name: Create Pull Request # creates a PR if there are differences
+      uses: peter-evans/create-pull-request@v4.1.1
+      id: cpr
+      with:
+        token: ${{ secrets.AUTOMATED_PR_GH_PAT }}
+        commit-message: Create new OpenEBS version
+        title: 'Automated OpenEBS version update ${{ steps.update.outputs.openebs_version }}'
+        branch: automation/update-openebs
```

Comment:
this should 100% be a different branch or else they'll force push over each other

Reply:
Thanks fixed